tests/python/gaia-ui-tests/gaiatest/form_controls/form_control.py

```diff
@@ -8,5 +8,8 @@
 
 class Widget(PageRegion):
     def __init__(self, marionette, locator):
-        element = Wait(marionette).until(expected.element_present(*locator))
+        if type(locator) is tuple:
```

Comment:
I agree, that would be the way we do in JavaScript. But in python, we can avoid to force a parameter in lieu of another:
```python
def __init__(self, marionette, locator=None, element=None):
   if locator is None and element is None:
        raise AttributeError('Either the locator or the element should be provided')
   if element is None:
        element =  Wait(marionette).until(expected.element_present(*locator))
   PageRegion.__init__(self, marionette, element)
```